docs/API_Reference.md

```diff
@@ -638,3 +643,259 @@ ehsm-core enclave will decrypt user-supplied ciphertextblob with specified CMK t
+
+- **Example**
+  - Request sample in python
+  ```python
+    params = OrderedDict()
+    params["appid"] = appid
+    params["timestamp"] = str(int(time.time() * 1000))
+
+    sign_string = urllib.parse.unquote(urllib.parse.urlencode(params))
+    sign = str(base64.b64encode(hmac.new(appkey.encode('utf-8'), sign_string.encode('utf-8'), digestmod=sha256).digest()),'utf-8').upper()
+
+    params["sign"] = sign
+    
+    requests.post(url="<ehsm_srv_address>/ehsm?Action=ListKey", data=json.dumps(params), headers=headers)
+  ```
+
+  - Response data
+  ```python
+    Response= {
+      "code": 200,
+      "message": "success!",
+      "result": {
+         "list": [
+             {
+                "keyid": "2de54366-30f9-6829-8391-c4eba***", 
+                "creationDate": 1645152619225, 
+                "expireTime": 1676688619225, 
+                "alias": "alias_name", 
+                "keyspec": "EH_RSA_3072", 
+                "keyState": 1
+             }
+         ]
+      }
+    }
+  ```
+  *(return to the [Cryptographic Functionalities APIs](#eHSM-REST-API-Reference).)*
+---
+
+## DeleteKey
+Delete a CMK.
+
+- **Rest API format:**
+
+  POST <ehsm_srv_address>/ehsm?Action=DeleteKey
+
+
+- **Request Payload:**
+
+  | Name | Type | Reference Value | Description |
+  |:-----------|:-----------|:-----------|:-----------|
+  | keyid | String | "2de54366-30f9-6829-8391-c4eba&lowast;&lowast;&lowast;" | ID of the CMK you want to delete |
+
+  Notes: for the common request parameters, please refer to the [common params](#Common-Prameters)
+
+- **Response Data:**
+
+  | Name | Type | Reference Value | Description |
+  |:-----------|:-----------|:-----------|:-----------|
+  | code | int | 200 | The result of the method call, 200 is success, others are fail. |
+  | message | String | "success" | The description of result. |
+
+- **Example**
+  - Request sample in python
+  ```python
+    payload = OrderedDict()
+    payload["keyid"] = "2de54366-30f9-6829-8391-c4eba***"
+
+    params = OrderedDict()
+    params["appid"] = appid
+    params["payload"] = urllib.parse.unquote(urllib.parse.urlencode(payload))
+    params["timestamp"] = str(int(time.time() * 1000))
+
+    sign_string = urllib.parse.unquote(urllib.parse.urlencode(params))
+    sign = str(base64.b64encode(hmac.new(appkey.encode('utf-8'), sign_string.encode('utf-8'), digestmod=sha256).digest()),'utf-8').upper()
+
+    params["payload"] = payload
+    params["sign"] = sign
+    
+    requests.post(url="<ehsm_srv_address>/ehsm?Action=DeleteKey", data=json.dumps(params), headers=headers)
+  ```
+
+  - Response data
+  ```python
+    Response= {
+      "code": 200,
+      "message": "success!",
+      "result": {}
+    }
+  ```
+  *(return to the [Cryptographic Functionalities APIs](#eHSM-REST-API-Reference).)*
+---
+
+## DeleteALLKey
+Delete all CMKs of the current appid.
+
```

Comment:
Delete all the CMKs generated by the current account.

Reply:
It will be fix